packages/core/user-profile/core-user-profile-browser/src/service.ts

```diff
@@ -17,10 +17,13 @@ import type {
 
 export interface UserProfileService {
   /**
-   * Retrieve an observable emitting when the user profile is loaded.
+   * Retrieve an observable emitting current user profile data.
```

Comment:
The comment was incorrect here. I created to improve impl https://github.com/elastic/kibana/issues/204570

Reply:
nit 😇 
```suggestion
   * Retrieve an observable emitting the current user profile data.
```